vnext/Scripts/installRNW.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @format
+ */
+// @ts-check
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const rimraf = require('rimraf');
+
+function retryOnError(errorCode, func) {
+  const startTime = Date.now();
+
+  while (true) {
+    try {
+      func();
+      return true;
+    } catch (ex) {
+      if (ex.code !== errorCode) {
+        throw ex;
+      }
+    }
+
+    if (Date.now() - startTime > 5000) {
+      return false;
+    }
+  }
+}
+
+function copyDirectories(srcPath, targetPath, dirSpecs) {
```

Comment:
This code is looking a little bit familiar 😉 . Can we share it with the above for the final version of this?

Reply:
sure, this is early code, just trying to get things working at this point.

---
In reply to: [405208226](https://github.com/microsoft/react-native-windows/pull/4522#discussion_r405208226) [](ancestors = 405208226)